terraform/060-simplesamlphp/main.tf

```diff
@@ -78,6 +79,7 @@ data "template_file" "task_def" {
     mysql_user                  = "${var.mysql_user}"
     recaptcha_key               = "${var.recaptcha_key}"
     recaptcha_secret            = "${var.recaptcha_secret}"
+    remember_me_secret          = "${var.remember_me_secret}"
```

Comment:
Do you also need to add `remember_me_secret` to the README.md file?

Reply:
oh yeah, guess so